filaments/devildesign.json

```diff
@@ -463,6 +463,228 @@
                     "hex": "005387"
                 }
             ]
+        },
+        {
+            "name": "{color_name}",
+            "material": "SILK",
```

Comment:
Material should be `"PLA"` and add a property `"finish"` with value `"glossy"`.

Reply:
Added